src/site/content/en/learn-web-vitals/crux-and-rum-differences/index.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: Why is CrUX data different from my RUM data?
+subhead:  |
+  Learn about reasons why RUM data can show different Core Web Vitals numbers from CrUX.
+description: |
+  Learn about reasons why RUM data can show different Core Web Vitals numbers from CrUX.
+authors:
+  - tunetheweb
+date: 2022-08-11
```

Comment:
```suggestion
date: 2022-08-12
```

Reply:
Will update this when launching. Will leave this comment open to remind me.